packages/tracing/src/browser/metrics.ts

```diff
@@ -170,6 +171,25 @@ export class MetricsInstrumentation {
       }
 
       transaction.setMeasurements(this._measurements);
+
+      if (this._lcpEntry) {
+        logger.log('[Measurements] Adding LCP Data');
+        // Capture Properties of the LCP element that contributes to the LCP.
+
+        if (this._lcpEntry.element) {
+          transaction.setTag('lcp.element', htmlTreeAsString(this._lcpEntry.element));
+        }
+
+        if (this._lcpEntry.id) {
+          transaction.setTag('lcp.id', this._lcpEntry.id);
+        }
+
+        if (this._lcpEntry.url) {
+          transaction.setTag('lcp.url', this._lcpEntry.url);
```

Comment:
I thought one curious case to consider is what if the image is not coming from a remote URL but instead inlined with a data URL, like 

```
<img src="data:image/gif;base64,R0lGODlhEAAOALMAAOazToeHh0tLS/7LZv/0jvb29t/f3//Ub/
/ge8WSLf/rhf/3kdbW1mxsbP//mf///yH5BAAAAAAALAAAAAAQAA4AAARe8L1Ekyky67QZ1hLnjM5UUde0ECwLJoExKcpp
V0aCcGCmTIHEIUEqjgaORCMxIC6e0CcguWw6aFjsVMkkIr7g77ZKPJjPZqIyd7sJAgVGoEGv2xsBxqNgYPj/gAwXEQA7" 
width="16" height="14" alt="embedded folder icon">
```

<img src="data:image/gif;base64,R0lGODlhEAAOALMAAOazToeHh0tLS/7LZv/0jvb29t/f3//Ub/
/ge8WSLf/rhf/3kdbW1mxsbP//mf///yH5BAAAAAAALAAAAAAQAA4AAARe8L1Ekyky67QZ1hLnjM5UUde0ECwLJoExKcpp
V0aCcGCmTIHEIUEqjgaORCMxIC6e0CcguWw6aFjsVMkkIr7g77ZKPJjPZqIyd7sJAgVGoEGv2xsBxqNgYPj/gAwXEQA7" 
width="16" height="14" alt="embedded folder icon">

Would we still want to include that as a tag? Definitely a corner case, but the image could be large enough that would cause the event/transaction to be dropped in Relay.

(I don't know if the browser performance API would ever report such URLs)

Reply:
I think we should include it. Ultimately the browser would determine what element contributes the LCP. IMO, if a data URL was marked as an LCP, then that might be a good signal that the page is performant. I think @k-fish (Kevan) can comment further.

Reply:
I don't know that it would provide a ton of value, except in cases where the url is necessary to discern between img tags, but the element tree should be sufficient in most cases to determine the exact lcp element. That being said, all we have to do to avoid having the transaction rejected is truncate to the relay tag value limit of `200` chars anyway.

Reply:
I tried this out. Inline images using a `data:` URI report the full URL in the Performance Entry. We don't want to send that, in particular not if it is too big.

![image](https://user-images.githubusercontent.com/88819/115706256-b944f900-a36d-11eb-86c9-b50b2758dc7b.png)

We could special-case `data:` and trim-off the contents (just indicate this was a `data URI`), since truncating won't be useful (cannot render an image from a truncated URI and the base64 blob makes no sense by itself other than string matching to something in the user's code base, but for that we'd have better tools like element path and/or id).

Reply:
@rhcarvalho Oh I see. I had forgotten that `data` URIs can be quite big. I can trim it to first `200` characters as suggested by @k-fish .

Reply:
@rhcarvalho should be addressed in https://github.com/getsentry/sentry-javascript/pull/3427/commits/c513a29e4c18e45d344aa7ee5332f706270a7bf9